### PR TITLE
EFRS-588 ADD drag and drop

### PR DESCRIPTION
**Pull request by mjakowiec**

### CODE REVIEW

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.scss [hunk 1]
```diff
@@ -0,0 +1,80 @@
+.dropzone {
+    width: 100%;
+    height: 200px;
+    text-align: center;
+    border: dashed 1px #979797;
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.scss:5]: #979797 is used a couple of times. All colors should be defined in colors.scss.

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.scss [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    position: relative;
+    margin: 0 auto;
+    background-color: lightgray;
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.scss:8]: should be sass variable from colors.scss

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      margin-bottom: 1rem;
+  
+      img.delete {
+        margin-left: 0.5rem;
+        cursor: pointer;
+        align-self: flex-end;
+      }
+  
+  
+      display: flex;
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.scss:57]: display flex already defined above

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      align-items: center;
+      border: dashed 1px #979797;
+      margin-bottom: 1rem;
+  
+      img.delete {
+        margin-left: 0.5rem;
+        cursor: pointer;
+        align-self: flex-end;
+      }
+
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.scss:55]: redundant empty line

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.scss [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  
+      img.delete {
+        margin-left: 0.5rem;
+        cursor: pointer;
+        align-self: flex-end;
+      }
+  
+  
+      display: flex;
+      flex-grow: 1;
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.scss:58]: move flex-grow to the top display: flex; definition

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.scss [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  
+      .name {
+        font-size: 14px;
+        font-weight: 500;
+        color: #353f4a;
+        margin: 0;
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.scss:64]: colors.scss

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.scss [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+      }
+  
+      .size {
+        font-size: 12px;
+        font-weight: 500;
+        color: #a4a4a4;
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.scss:70]: colors.scss

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.ts [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+export class DragNDropComponent {
+
+  @ViewChild('fileDropRef', { static: false }) fileDropEl: ElementRef;
+  file: any;
+
+  /**
+   * on file drop handler
+   */
+  onFileDropped($event) {
+    console.log('File upload');
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.ts:17]: remove console.log

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.ts [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  /**
+   * Process file
+   * @param files (Files List)
+   * TODO Send file to api
+   */
+  processFile(files: Array<any>) {
+    for (const item of files) {
+      this.file = item;
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.ts:35]: this.file will always be the last item in the files array

File: ui/src/assets/dnd/tmp-upload-file.svg [hunk 10]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="63" height="64" viewBox="0 0 63 64">
```
Comment (isobolev) [inline ui/src/assets/dnd/tmp-upload-file.svg:1]: the file name is starting with temp. Did you confirm this image with Sergey?

**Review by isobolev — COMMENTED** [review]
good job, minor comments to resolve

File: ui/src/assets/dnd/tmp-upload-file.svg (hunk 10 above)
Comment (mjakowiec) [reply ui/src/assets/dnd/tmp-upload-file.svg:1]: No, i use this image as placeholder

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.scss (hunk 2 above)
Comment (isobolev) [reply ui/src/app/features/drag-n-drop/drag-n-drop.component.scss:8]: please resolve

**Review by isobolev — APPROVED** [review]

**Pull request merged**